Gemfile

```diff
@@ -2,8 +2,17 @@
 
 source "https://rubygems.org"
 
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
```

Comment:
What's this used for?

Reply:
This ensures https is used when pulling in dependencies from a tag, a branch, or master rather than a gem release, which I was initially doing with curb, but not now. Removed.